README.md

```diff
@@ -45,15 +46,91 @@ def sayHello:
 
 > Note that we cannot support tooltips without a VSCode extension due to behavior of the [static analyzer](https://github.com/davidhalter/jedi/issues/1921) used in VSCode.
 
+## Alerts / SLOs
+
+Autometrics makes it easy to add Prometheus alerts using Service-Level Objectives (SLOs) to a function or group of functions.
+
+This works using pre-defined [Prometheus alerting rules](https://github.com/autometrics-dev/autometrics-shared#prometheus-recording--alerting-rules). By default, most of the recording rules are dormant. They are enabled by specific metric labels that can be automatically attached by autometrics.
```

Comment:
users need to add the autometrics alerting rules to their prometheus, right? maybe we could reiterate that here

Reply:
👍 I copied this over from the rust library, so we should probably update it there as well

Reply:
cool i opened a PR for that https://github.com/autometrics-dev/autometrics-rs/pull/63